### PR TITLE
Fix the issues mentioned in the PR to mopro/main

### PR DESCRIPTION
**Pull request by moven0831**
https://github.com/oskarth/mopro/pull/53

### CODE REVIEW

File: mopro-ffi/src/mopro.udl
Comment (FoodChain1028) [inline mopro-ffi/src/mopro.udl:1]: Is this completed yet? 

Or is there any specific reason to comment these lines?

**Review by FoodChain1028 — COMMENTED** [review]
LGTM except udl part.

File: mopro-ffi/src/mopro.udl
Comment (moven0831) [reply mopro-ffi/src/mopro.udl:1]: Done so far! Thanks for checking

**Review by FoodChain1028 — APPROVED** [review]
LGTM!

**Pull request merged**